Malik.9892.token sell from explore page

mzparacha opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9892

## Description of Changes
- Adds FIXME widget to TODO page.
- 

## "How We Fixed It"
<!-- Brief description of solution, if bug, to be added once issue is resolved. -->

## Test Plan
- Unit tested the `FIXME()` call.
- CA (click around) tested on local and frack:
  - TODO page
  - 

## Deployment Plan
<!--- Omit if unneeded -->
1. 

## Other Considerations
<!--- Follow-up tickets, breaking changes, etc -->
-

masvelio commented on packages/commonwealth/client/scripts/state/api/launchPad/tokenEthExchangeRate.ts:
```diff
@@ -0,0 +1,59 @@
+        tokenAmount,
+        mode,
+      }),
+    staleTime: 1000, // 1 second
```
can you move it to const

masvelio commented on packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/useBuyTrade.ts:
```diff
@@ -0,0 +1,193 @@
+      const payload = {
+        chainRpc: chainNode.url,
+        ethChainId: chainNode.ethChainId,
+        amountEth: ethBuyAmount * 1e18, // amount in wei
```
@rbennettcw is this a case for using big int?

mzparacha replied:
I discussed with Ian and we need to convert buy amounts to bigint for passing to helper. Resolving this for merge.

masvelio commented on packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/useTradeTokenForm.ts:
```diff
@@ -198,40 +112,31 @@ const useTradeTokenForm = ({
-        eth: commonPlatformFeeForBuyTradeInEth,
-      },
     },
+    // TODO: add presets for max amounts?
```
?

masvelio commented on packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/AmountSelections/SellAmountSelection.tsx:
```diff
@@ -0,0 +1,52 @@
+  return (
+    <div className="AmountSelections">
+      <div className="amount-input-with-currency-symbol">
+        <CWTextInput
```
shouldn't we make it type number?

mzparacha replied:
originally it was, those float values were used in calculations and needed to be displayed differently. So I settled with string storage and float parsing those values

mzparacha commented on packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/AmountSelections/SellAmountSelection.tsx:
```diff
@@ -0,0 +1,52 @@
+        <CWTextInput
+          containerClassName="amount-input"
+          placeholder="0"
+          value={trading.amounts.sell.invest.baseToken.amount}
```
yes

https://github.com/hicommonwealth/commonwealth/blob/6c86fb5f1a7ab0d7162dcc16c37eed45c0791738/packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/useSellTrade.ts#L22-L23

https://github.com/hicommonwealth/commonwealth/blob/6c86fb5f1a7ab0d7162dcc16c37eed45c0791738/packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/useSellTrade.ts#L33

https://github.com/hicommonwealth/commonwealth/blob/6c86fb5f1a7ab0d7162dcc16c37eed45c0791738/packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/useSellTrade.ts#L65-L81

mzparacha commented:
Re: https://github.com/hicommonwealth/commonwealth/pull/9926#pullrequestreview-2442368693

1. query wasn't getting recalled even tho I added cache invalidation. fixed now.
2. the hidden scrollbar was causing 8px diff, fixed now.
3. fixed.
4. same as 1, query needed a refresh, fixed now.

Thanks for the review @masvelio
